doc/rose-rug-suite-writing-tutorial.html

```diff
@@ -787,6 +787,10 @@ <h3 class="alwayshidden">Visualising the Data</h3>
 
       <p>You'll now get a useful link in your navigate task output. Expect a
       wiggly course!</p>
+
+      <p>Note: If you are viewing the job log via Rose Bush, try switching on
+      Tags mode by clicking on the <kbd>Tags</kbd> at the top right hand corner
+      of the page.</p>
```

Comment:
This is some what related to #1809. I think our version of Jinja2's `urlize` wasn't converting the link properly.